docs/source/workflow-apis.md

```diff
@@ -0,0 +1,170 @@
+# Workflow APIs
+
+## Using Workflow APIs
+
+To start using a Workflow API you have to know which API you are going to be using.
+In this case we are assuming that the repository defines a v2 namespace with an API definition of MyExampleAPI.
+
+Now to use it, you have to import the API class object you want to use and specify the type of the class in the `apis`
+field.
+This is necessary so all the messages that are consumed and produced by the API can be picked up by the framework
+and the actor can depend on them.
+
+Here is an example of the usage:
+
+```python
+from leapp.actors import Actor
+from leapp.workflows.api.v2 import MyExampleAPI
+
+class MyWorkflowAPIUsingActor(Actor):
+    """ An example actor consuming the v2.MyExampleAPI Workflow API """
+    produces = ()
+    consumes = ()
+    apis = (MyExampleAPI,)
+
+    def process(self):
+        api = MyExampleAPI()
+        api.second(value=10)
+```
+
+## General workings of the Workflow API feature
+
+In the leapp repository , you want to create the API, for create a folder called `apis`.
```

Comment:
The sentence has probably typo or it's incomplete. And use please *directory* instead of folder :)